src/nerve/gateway/anthropic_proxy.py

```
@@ -238,17 +312,115 @@ async def _handle_streaming(
+            # Normal mode: use aiohttp
+            await self._handle_streaming_aiohttp(
+                response, body, url, trace_id, forward_headers, debug_events
+            )
+
+        # Save debug events
+        self._save_debug(trace_id, "2_response_events.json", debug_events)
+
+        try:
+            await response.write_eof()
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)